packages/commons/tests/unit/typeUtils.test.ts

```diff
@@ -119,6 +120,32 @@ describe('Functions: typeUtils', () => {
     });
   });
 
+  describe('Function: isStringUndefinedNullEmpty', () => {
+    it('should return true if input is undefined', () => {
```

Comment:
```suggestion
    it('returns true if input is undefined', () => {
```

Reply:
Can we apply this to all the tests, and also add a comment `// Act & Assess` on top of each call?